src/commands/Admin/reload.js

```diff
@@ -39,4 +40,18 @@ module.exports = class extends Command {
 		}
 	}
 
+	async all(message) {
+		const timer = new Stopwatch();
+		await Promise.all(this.client.pieceStores.map(async (p) => {
+			await p.loadAll();
+			await p.init();
+		}));
+		if (this.client.shard) {
+			await this.client.shard.broadcastEval(`
+				if (this.shard.id !== ${this.client.shard.id}) this.client.pieceStores.map(async (p) => await p.loadAll());
+			`);
+		}
+		return message.sendMessage(`Reloaded all stores. (Took: ${timer.stop().toString()})`);
```

Comment:
toString() is unnecessary in a template literal, it already implicitly calls toString();

Reply:
Noted, will push a change about this.